packages/qdrant-loader-core/tests/test_gemini_provider.py

```
@@ -0,0 +1,361 @@
+            state["client_kwargs"] = kwargs
+            self.models = _Models()
+
+    genai_mod = types.ModuleType("google.genai")
+    genai_mod.Client = Client
+    genai_mod.types = types_mod
+    genai_mod.errors = errors_mod
+
+    return genai_mod, types_mod, errors_mod, state
+
+
+def _reload_gemini(monkeypatch, **kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_reload_gemini`

(ANN202)